app/views/createAccount/registerOrganisation.scala.html

```diff
@@ -132,12 +132,12 @@ <h1 class="govuk-heading-l">@messages("enrolment.title")</h1>
             name = "isAgent",
             items = Seq(
                 RadioItem(
-                    content = Text("Yes"),
+                    content = Text(messages("app.common.yes")),
                     value = Some("true"),
                     checked = form.data.get("isAgent").fold(false)(_ == "true")
                 ),
                 RadioItem(
-                    content = Text("No"),
+                    content = Text(messages("app.common.no")),
                     value = Some("false"),
                     checked = form.data.get("isAgent").fold(true)(_ == "false")
                 )
```

Comment:
Could we put a comment in JIRA on the story please to say that this issue has been fixed has part of the ticket, just in case it get's raised by anyone else at another point.